aws/data_source_aws_iam_group.go

```diff
@@ -30,6 +30,30 @@ func dataSourceAwsIAMGroup() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"members": {
```

Comment:
Just noting here, that we may want to consider naming this attribute `users` to match the API. 👍

Reply:
Thanks, i fixed.